pom.xml

```diff
@@ -175,6 +191,95 @@
       </pluginManagement>
 
         <plugins>
+            <plugin>
+              <groupId>org.openapitools</groupId>
+              <artifactId>openapi-generator-maven-plugin</artifactId>
+              <version>${openapi-generator.version}</version>
+              <executions>
+                <execution>
+                  <id>generate-api-and-models</id>
+                  <phase>generate-sources</phase>
+                  <goals>
+                    <goal>generate</goal>
+                  </goals>
+                  <configuration>
+                    <inputSpec>
+                      ${project.basedir}/src/main/resources/META-INF/openapi.yml
+                    </inputSpec>
+                    <output>${project.build.directory}/generated-sources/openapi-spec</output>
+
+                    <skipValidateSpec>true</skipValidateSpec>
+
+                    <skipIfSpecIsUnchanged>false</skipIfSpecIsUnchanged>
+                    <cleanupOutput>false</cleanupOutput>
+
+                    <generatorName>jaxrs-spec</generatorName>
+                    <templateDirectory>${project.basedir}/src/main/openapi-generator/JavaJaxRs/spec</templateDirectory>
+
+                    <globalProperties>
+                      <skipFormModel>false</skipFormModel>
+                    </globalProperties>
+
+                    <packageName>com.redhat.composer</packageName>
+                    <invokerPackage>com.redhat.composer</invokerPackage>
+                    <apiPackage>com.redhat.composer.api</apiPackage>
+                    <modelPackage>com.redhat.composer.api.model</modelPackage>
+
+                    <generateAliasAsModel>true</generateAliasAsModel>
+
+                    <generateApis>true</generateApis>
+                    <generateApiTests>false</generateApiTests>
+                    <generateApiDocumentation>false</generateApiDocumentation>
+                    <generateModels>true</generateModels>
+                    <generateModelTests>false</generateModelTests>
+                    <generateModelDocumentation>false</generateModelDocumentation>
+                    <generateSupportingFiles>false</generateSupportingFiles>
+
+                    <!-- Don't include multipart APIs, since openapi-generator doesn't handle multipart correctly with jaxrs-spec yet -->
+                    <apisToGenerate>Assistant,AssistantAdmin,LlmConnectionAdmin,ContentRetrieverConnectionAdmin</apisToGenerate>
```

Comment:
Hmm interesting, so we will need to make sure we keep up with the spec here.

General note (not for this pr). We will need to make sure how to update the contract is well documented.